contrib/tvmop/compile.py

```diff
@@ -152,6 +152,12 @@ def get_cuda_arch(arch):
     # we create libtvmop.o first, which gives us chance to link tvm_runtime together with the libtvmop
     # to allow mxnet find external helper functions in libtvm_runtime
     func_binary.save(arguments.target_path + "/libtvmop.o")
+    try:
+        func_binary.imported_modules
+    except NameError:
+        func_binary.imported_modules = []
```

Comment:
from https://github.com/apache/incubator-tvm/blob/master/python/tvm/runtime/module.py#L136 we can see `func_binary.imported_modules` should always exist.

Reply:
Thank you for your review. I have deleted these lines.